launchable/utils/session.py

```diff
@@ -0,0 +1,60 @@
+import click
+import os
+import json
+from pathlib import Path
+import shutil
+from typing import Optional
+import datetime
+
+SESSION_DIR_KEY = 'LAUNCHABLE_SESSION_DIR'
+DEFAULT_SESSION_DIR = '~/.config/launchable/sessions/'
+
+
+def _session_file_dir() -> Path:
+    return Path(os.environ.get(SESSION_DIR_KEY) or DEFAULT_SESSION_DIR).expanduser()
+
+
+def _session_file_path(build_name: str) -> Path:
+    return _session_file_dir() / "{}:{}.txt".format(build_name, os.getsid(os.getpid()))
```

Comment:
Just one last thing, hopefully. build name is a user supplied string and there's no guarantee that it's safe as a file name.

Different ways to solve this, but my suggestion is to hash the build name.